lib/config.py

```diff
@@ -34,6 +34,9 @@
 GENERATE_DEGREE_ANAL = False
 PRINT_CHANNEL_USER_HASH = False
 
+#MessageTimeGraph and NickChange constants
```

Comment:
this is not needed. we have included this in function definition.
check the function definitions if messageTime and NickChange